index.html

```diff
@@ -8,32 +8,200 @@
-        <header id="header">
-            <h1 class="logo"><a href="">My Logo</a></h1>
-            
-            <nav class="nav_icon">
-                <a href="#"><img src="./images/menu-icon.png" alt="menu-icon"></a>
-            </nav>
-        </header>
-        <section class="page-one" id="first_page">
-    <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">Glad to see <br> you!</span></h1>
-    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
-        work and experience! If you like what you see and have a project you need coded, don’t hestiate to
-        contact me.</p>
-    
-        <div class="connect">
-            Let's connect
+    <!-- first page -->
+    <section class="cover">
+        <div class="header_margin">
+            <header id="header">
+                <h1 class="logo"><a href="">My Logo</a></h1>
+                <nav class="nav_icon">
+                    <a href="#"><img src="./images/menu-icon.png" alt="menu-icon"></a>
+                </nav>
+            </header>
+            <section class="page-one" id="first_page">
+                <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">Glad to see <br> you!</span></h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+                <div class="connect"><a href="#">Let's connect.</a>
+                </div>
+                <div class="links">
+                    <ul>
+                        <li><a href="#"><img src="./images/twitter.svg" alt="twiter_icon"></a></li>
+                        <li><a href="#"><img src="./images/linkedin.svg" alt="linkedin_icon"></a></li>
+                        <li><a href="#"><img src ="./images/medium.svg" alt="medium_icon"></a></li>
+                        <li><a href="#"><img src="./images/github.svg" alt="github_icon"></a></li>
+                        <li><a href="#"><img src="./images/angellist.svg" alt="angellist_icon"></a></li>
+                    </ul>
+                </div>
+            </section>
+        </div>
+    </section>
+
+    <section id="mid">
+        <!-- second page -->
+         <div class="left-block">
+            <div class="primary-image">
+                <img src="images/project1.png" alt="photo_of_my_projects">
+            </div>
+            <div class="title">
+                <h1>Tonic</h1>
+            </div>
+            <div class="frame-2">
+                <ul>
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="primary-text"> Daily selection of privately personalised reads" no accounts or signups required. </div>
```

Comment:
- [ ] **[REQUIRED]** This was mentioned by the previous reviewer, please change the `div` here to a `p` tag as this will improve the page accessibility to assistive technology.  Please do the same for all the cards. 

it should be `<p class="primary-text"> Daily selection of privately personalised reads" no accounts or signups required. </p>` and not `<div class="primary-text"> Daily selection of privately personalised reads" no accounts or signups required. </div>`